include/fmt/format.h

```diff
@@ -1794,7 +1798,8 @@ class specs_handler : public specs_setter<typename Context::char_type> {
   typedef typename Context::format_arg format_arg;
 
   FMT_CONSTEXPR format_arg get_arg(auto_id) {
-    return internal::get_arg(context_, parse_context_.next_arg_id());
+    return internal::get_arg(
+        context_, static_cast<unsigned>(parse_context_.next_arg_id()));
```

Comment:
Why is it needed? `next_arg_id()` already returns `unsigned`.

Reply:
This was to suppress a warning I was getting with [clang-trunk](https://i.imgur.com/rd2U6Np.png) (-Wshorten-64-to-32)

Reply:
I couldn't repro this: https://godbolt.org/z/tfq3_w. Could you provide an example that reproduces this warning?

Reply:
Can't seem to reproduce it on godbolt, but the warning went away when I included `fmt/format.h` instead of `format` - I was getting this warning with just a call to `fmt::print`.